specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2015-03-01-preview/applications.json

```diff
@@ -233,7 +233,14 @@
         "accessModes": {
           "type": "array",
           "items": {
-            "type": "string"
+            "type": "string",
+            "enum": [
+              "WebPage"
```

Comment:
There's only one possible value for this enum?

Reply:
> There's only one possible value for this enum?

Yes. At least for now, there's only one possible value for this enum.